### DISCUSSION TITLE
How to set to auto start web application on IIS ?

### DISCUSSION THREAD

**Discussion by lakshmi-madhushika** [community/community#140870] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

There was a issue to set to auto start application on iis and then create web api log without browsing . 

1. first we should set application pool AlwaysRunning and Idle Time-Out=0
2. next change site preload enable=true. in here if you use sub application , we should change preload enable=true for this sub application.
![image](https://github.com/user-attachments/assets/249f395c-967c-42fd-a104-704ffd929ebe)

![image](https://github.com/user-attachments/assets/ef1959c5-226c-401d-a0bf-b02924b1f0dc)

![image](https://github.com/user-attachments/assets/47bda2eb-8002-400d-9f9c-3110c2068a82)

3. at last restart aplication pool and site and check log path

**Comment by LiteBrite82** [comment]
Hi @lakshmi-madhushika ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
